manifests/plugin/processes.pp

```diff
@@ -17,8 +17,14 @@
     interval => $interval,
   }
 
+  if ( $processes or $processes_matches ) {
```

Comment:
this is giving us the error:
```
manifests/plugin/processes.pp:20:variable_scope:WARNING:top-scope variable being used without an explicit namespace
```

which is confusing… did you typo?
doesn't look like a typo to me :S